src/sentry/search/utils.py

```diff
@@ -280,28 +290,30 @@ def get_date_params(value, from_field, to_field):
     if date_to is not None:
         date_to_value, date_to_inclusive = date_to
         result.update({to_field: date_to_value, f"{to_field}_inclusive": date_to_inclusive})
-    return result
+    return result  # type: ignore[return-value]
```

Comment:
`result` is built entirely in this function, how is it possible that the typing could be wrong?

Reply:
Ahh, just needed to define the type for the `result`